qgis-samgeo-plugin/samgeo_plugin.py

```
@@ -1379,6 +1618,135 @@ def segment_by_points_batch(self):
         finally:
             self.progress_bar.setVisible(False)
 
+    def _auto_show_results(self):
```

Ruff (0.16.1)

[warning] Too many branches (16 > 12)

(PLR0912)